react/components/Charts/LineChart/constants.ts

```diff
@@ -0,0 +1,15 @@
+import { LineProps } from 'recharts'
+const colors = [
+    '#EE7850',
+    '#4CAABF',
+    '#9F50B6',
+    '#479A5E'
+]
+type DefaultLineProps = Pick<LineProps, 'type' | 'strokeWidth' | 'dot'>
+const defaultProps: DefaultLineProps = {
+    type: 'monotone',
```

Comment:
I think you should use the 'linear' type

Reply:
Following the figma, it seems that is `monotone`, check it out: https://www.figma.com/file/G2jbai2bVLw1u7JOSbw6R9/02-Charts